### PR TITLE
Add Castle theme to Lapce

### PR DESCRIPTION
**Pull request by ionutrogojan**
Thank you :)

### CODE REVIEW

File: volts2 [hunk 1]
```diff
@@ -222,5 +222,12 @@
         "author": "VarLad",
         "description": "Julia plugin for Lapce",
         "meta": "https://raw.githubusercontent.com/VarLad/lapce-julia/master/volt.toml"
+    },
+    {
+        "name" = "castle",
+        "version": "0.1.0",
+        "display-name" = "Castle",
+        "author" = "Ionut Rogojan",
```
Comment (dzhou121) [inline volts2:230]: This ```=``` should be ```:```

File: volts2 [hunk 2]
```diff
@@ -222,5 +222,12 @@
         "author": "VarLad",
         "description": "Julia plugin for Lapce",
         "meta": "https://raw.githubusercontent.com/VarLad/lapce-julia/master/volt.toml"
+    },
+    {
+        "name" = "castle",
+        "version": "0.1.0",
+        "display-name" = "Castle",
```
Comment (dzhou121) [inline volts2:229]: This ```=``` should be ```:```

File: volts2 [hunk 3]
```diff
@@ -222,5 +222,12 @@
...
+        "author" = "Ionut Rogojan",
+        "meta": "https://raw.githubusercontent.com/ionutrogojan/castle/main/plugin.toml"
```
Comment (dzhou121) [inline volts2:231]: You're missing ```description```

**Comment by ionutrogojan** [comment]
Hi, sorry about that. I copied some of the values from the `.toml` file and forgot to update the syntax. It should be correct now. :)

File: volts2 (hunk 1 above)
Comment (ionutrogojan) [reply volts2]: fixed

**Pull request merged**